### PR TITLE
Feat testing

### PR DESCRIPTION
**Pull request by omar-labana**
Highlights on this pull request:
- Add tests for calculator logic making sure each calculation works
- Add tests to the button component along with Home view, Calculator view, and Quote view

### CODE REVIEW

File: src/test/components/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  it('exists', () => {
+    expect(wrapper.find('Calculator')).toBeTruthy();
+  });
+
+  it('has to match snap', () => {
+    const calc = renderer.create(<Calculator />);
+    expect(calc).toMatchSnapshot();
+  });
+});
```
Comment (george-shammar) [inline src/test/components/Calculator.test.js:18]: - [ ] Please include instructions on how to run your tests in the README.
- [ ] Below is the output of the tests from your application. Please let's fix it. As mentioned above, it'll be nice to explain any exceptional configuration for your tests beyond the usual installation of dependencies and running the tests.

test

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @omar-labana , 

Good job so far!

:heavy_check_mark: Your repository looks professional.

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/test/components/Calculator.test.js (hunk 1 above)
Comment (omar-labana) [reply src/test/components/Calculator.test.js:18]: Hey George, I ran all the tests and everything seems to be working fine
image

**Review by rukundoeric — APPROVED** [review]
Hi @omar-labana,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**